sass/_cover.scss

```diff
@@ -1,138 +1,134 @@
 .cover {
 	background-color: $background-color-cover;
-    color: #fff;
-    overflow: hidden;
-    padding: 156px 0 80px;
-    position: relative;
+	color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.